lib/constants.js

```diff
@@ -18,3 +18,4 @@ module.exports.API_DATASET_FEATURE = '/datasets/v1/{owner}/{dataset}/features/{i
 module.exports.API_TILESTATS_STATISTICS = '/tilestats/v1/{owner}/{tileset}';
 module.exports.API_TILESTATS_LAYER = '/tilestats/v1/{owner}/{tileset}/{layer}';
 module.exports.API_TILESTATS_ATTRIBUTE = '/tilestats/v1/{owner}/{tileset}/{layer}/{attribute}';
+module.exports.API_STATIC = '/v4/{mapid}{+overlay}/{+xyz}/{width}x{height}{+retina}{.format}';
```

Comment:
this url segment can be x,y,z or it can be `auto`, so i made it one chunk instead of three